hoot-core/src/main/cpp/hoot/core/io/HootApiDbBulkInserter.cpp

```diff
@@ -498,17 +498,17 @@ void HootApiDbBulkInserter::writePartial(const ConstRelationPtr& relation)
   const unsigned long relationDbId = _establishIdMapping(relation->getElementId());
   LOG_VART(ElementId(ElementType::Relation, relationDbId));
 
+  Tags tags = relation->getTags();
+  //  keep the hoot:id tag in sync with what could be a newly assigned id
   if (_includeDebugTags)
-  {
-    Tags tags = relation->getTags();
-    //keep the hoot:id tag in sync with what could be a newly assigned id
     tags.set(MetadataTags::HootId(), QString::number(relationDbId));
-  }
+  if (relation->getType() != "")
+    tags["type"] = relation->getType();
```

Comment:
Add the relation `type` tag back in from its place as a member variable of the `relation` class.  Don't want to lose that data because it defaults back to `multilinestring` which isn't accurate.